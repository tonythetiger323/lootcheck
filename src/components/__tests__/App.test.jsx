import React from "react";
import {shallow} from "enzyme";
import App from "../App";

describe("App component", () => {
  const app = shallow(<App />);

  it("renders properly", () => {
    expect(app).toMatchSnapshot();
  });

  it("contains a wallet component", () => {
    expect(app.find("Connect(Wallet)").exists()).toBe(true);
  })
});
