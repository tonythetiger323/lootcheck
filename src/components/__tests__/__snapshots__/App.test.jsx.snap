// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders properly 1`] = `
<div>
  <h2>
    Loot Check
  </h2>
  <hr />
  <Connect(Wallet) />
  <hr />
  <Connect(Loot) />
  <div>
    Powered by
     
    <a
      href="https://www.coindesk.com/price"
      target="_blank"
    >
      Coindesk
    </a>
  </div>
</div>
`;
